jina/drivers/__init__.py

```diff
@@ -134,15 +137,27 @@ class BaseDriver(metaclass=DriverType):
 
     store_args_kwargs = False  #: set this to ``True`` to save ``args`` (in a list) and ``kwargs`` (in a map) in YAML config
 
-    def __init__(self, priority: int = 0, *args, **kwargs):
+    def __init__(self, priority: int = 0, expect_parts: Optional[int] = 1, *args, **kwargs):
```

Comment:
Remember to add docstring for num parts

Reply:
Now a user needs to know how many parts it expects? It seems worse than before when user did not have to worry

Reply:
no need, but this is for preparing #1215